README.md

```diff
@@ -1,52 +1,63 @@
+rosdep update --rosdistro $ROS_DISTRO
+rosdep install -i --from-path src --rosdistro $ROS_DISTRO -y
+catkin build
+```
 
+<!-- ## Usage
 
-### Using different wheel types
+Basic Panther configuration can be found in file [panther.urdf.xacro](./panther_description/urdf/panther.urdf.xacro). This is an example configuration showing how to use the model. This can be used to import in launch files as a baseline model. For more advanced use cases, [panther_macro.urdf.xacro](./panther_description/urdf/panther_macro.urdf.xacro) is designed to be integrated into custom robot configurations. -->
 
-Wheel type is determined by `wheel_props_path` parameter. Those YAML files define all physical properties of wheels and their gazebo plugin.
+## Parameters
 
-Predefined wheels:
-- `WH01.yaml` - off-road skid drive wheels. Panther standard wheels
+Arguments passed to the [panther.urdf.xacro](./panther_description/urdf/panther.urdf.xacro) are the same as parameters of [panther_macro.urdf.xacro](./panther_description/urdf/panther_macro.urdf.xacro). Thus, this section covers both of them.
 
-- `WH02.yaml` - mecanum wheels 
 
-- `WH04.yaml` - small skid drive wheels
+- `use_sim` *(default: false)* - Changes between *ros2_control* for simulation and hardware **[TO BE IMPLEMENTED]**.
+- `dual_bat` *(default: false)* - Changes inertia and mass for robot body to match 2 batteries setup.
+- `imu_pos_x` *(default: 0.169)* - **x** coordinate of IMU sensor in relation to `body_link`.
+- `imu_pos_y` *(default: 0.025)* - **y** coordinate of IMU sensor in relation to `body_link`.
+- `imu_pos_z` *(default: 0.092)* - **z** coordinate of IMU sensor in relation to `body_link`.
+- `imu_rot_r` *(default: 0.0)* - roll rotation of IMU sensor in relation to `body_link`.
+- `imu_rot_p` *(default: 0.0)* - pitch rotation of IMU sensor in relation to `body_link`.
+- `imu_rot_y`  *(default: 0.0)* - yaw rotation of IMU sensor in relation to `body_link`.
+- `wheel_config_path` *(default: $(find panther_description)/config/WH01.yaml)* - absolute path to YAML file defining wheel properties.
+- `simulation_engine` *(default: gazebo-classic)* - physics engine to select plugins for. Supported engines: `gazebo-classic` **[MORE ENGINES TO BE IMPLEMENTED]**.
 
-### Changing sensor configuration
+There is one additional [panther.urdf.xacro](./panther_description/urdf/panther.urdf.xacro) argument:
+- `use_gpu` *(default: false)* - Turns on GPU acceleration for sensors.
+It is not present in the [panther_macro.urdf.xacro](./panther_description/urdf/panther_macro.urdf.xacro) since base of a robot does not have any sensors that can be accelerated with GPU.
 
-Predefined sensors are:
-- Orbbec Astra
-- Slamtec RPLIDAR S1
-- Velodyne Puck
-- Ouster OS1 32
+Parameter `wheel_config_path` allows using non-standard wheels with Panther robot without modifying URDF file. Syntax is following:
+- `wheel_radius` - wheel radius.
+- `wheel_separation` - separation of wheels alongside *y* axis.
```

Comment:
Maybe also add units here (they are specified in other parameters)?

Reply:
Done